custom_components/opnsense/__init__.py

```
@@ -258,12 +259,26 @@
 
-    @property
-    def coordinator_context(self):
-        return None
+    _client: OPNsenseClient = None
+
+    def __init__(
+        self,
+        config_entry: ConfigEntry,
+        coordinator: OPNsenseDataUpdateCoordinator,
+        unique_id_suffix: str,
+        name_suffix: str = None,
```

Ruff

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)